mentors/ruby/d.json

```
@@ -46,4 +46,12 @@
     "avatar_url": "https://avatars1.githubusercontent.com/u/89930?s=460&v=4",
     "bio": "I love Ruby for its beauty and consistency. I've been happily working with it for 10+ years and don't see myself stopping any time soon. Mentoring has been a great way to help the community while reinforcing my skills and knowledge in programming."
   }
```

```suggestion
  },
```
Need a comma to separate entries.

---

Thanks!